pkg/kube/pod_controller_test.go

```diff
@@ -131,7 +130,8 @@ func (s *PodControllerTestSuite) TestPodControllerWaitPod(c *C) {
 			c.Assert(pcp.InWaitForPodReadyPodName, Equals, podControllerPodName)
 			c.Assert(pcp.InWaitForPodReadyNamespace, Equals, podControllerNS)
 			c.Assert(errors.Is(err, pcp.WaitForPodReadyErr), Equals, true)
-			c.Assert(err.Error(), Equals, fmt.Sprintf("Pod failed to become ready in time: %s", simulatedError.Error()))
+
+			c.Assert(err.Error(), Equals, "Pod failed to become ready in time: SimulatedError")
```

Comment:
Should we add additional details `simulatedError.Error()` in the message or field?

Reply:
Parameters are not propagated to the error text. However, I've modified the test to simulate a wrapped error now.